package.json

```diff
@@ -38,6 +38,7 @@
     "url-loader": "^0.6.2",
     "webpack": "^3.6.0",
     "webpack-dev-server": "^2.9.1",
-    "webpack-merge": "^4.1.0"
+    "webpack-merge": "^4.1.0",
+    "http-server": "^0.10.0"
```

Comment:
did you maybe forge to use `yarn add` here?  Usually the dependencies are listed alphabetically.

Reply:
Yeah, I forgot the dev flag when adding it, so I just pasted it onto the end. I'll avoid that in the future though.

Reply:
no worries!  Do you mind moving it into the right spot in the list?  thanks!